.github/ISSUE_TEMPLATE/mystery_event.yml

```diff
@@ -20,26 +20,39 @@ body:
     id: rarity
     attributes:
       label: Rarity Tier
-      description: Check out the [Event Proposal Guide](https://github.com/AsdarDevelops/PokeRogue-Events/blob/mystery-battle-events/MEs_Proposal_Guide.md#do-tiers-mean-anything) if you have not yet!
+      description: Check out the [Event Proposal Guide](https://github.com/AsdarDevelops/PokeRogue-Events/blob/mystery-battle-events/MEs_Proposal_Guide.md) if you have not yet!
       multiple: false
       options:
         - Common
         - Great
         - Ultra
         - Rogue
+        - Part of a "Quest"
         - Other or unsure (please specify)
     validations:
       required: true
   - type: markdown # SEPARATOR
     attributes:
       value: |
         ---
```

Comment:
I would remove the separator here since they belong together

Reply:
You can add suggestions with "ctrl + g" or pressing this button on the right:

![image](https://github.com/user-attachments/assets/ef702dd7-5bc4-47bb-b56b-675aee3b40dd)

So that I just press "commit suggestion" and is done. But yes it's a good idea to remove it.

Reply:
Ik but I was on mobile